serveradmin/servershell/static/js/servershell/autocomplete/search.js

```diff
@@ -14,85 +14,111 @@ $(document).ready(function () {
         return result;
     };
 
-    let term_input = $('#term');
-    term_input.autocomplete({
-        delay: 150, // Wait n ms before starting to auto complete to avoid needles requests to backend
+    let _is_attribute_value = function(to_complete) {
+        /**
+         * Check if attribute value
+         *
+         * Check if to_complete is attribute value like for example a=b and
+         * if yes return attribute and value otherwise false.
+         *
+         * @type {Array|Boolean}
+         */
+        let match = to_complete.match(/([a-z_]+)=([\S]+)?/);
+
+        if (match) {
+            let attribute = match[1];
+            let value = match[2] === undefined ? '' : match[2];
+
+            return [attribute, value];
+        }
+
+        return false;
```

Comment:
I would go for early return to make the actual code to be on the "root-indention":
```suggestion
        if (!match) {
            return false;
        }

        let attribute = match[1];
        let value = match[2] === undefined ? '' : match[2];

        return [attribute, value];
```